### PR TITLE
Comments & import indicators added

### PR DESCRIPTION
**Pull request by steven004**
Studying the EC implementation, and have some handy comments, or fix invalid reference.

### CODE REVIEW

File: consensus/expected.go [hunk 1]
```diff
@@ -8,12 +11,12 @@ import (
 	"math/big"
 	"strings"
 
-	"gx/ipfs/QmNf3wujpV2Y7Lnj2hy2UrmuX8bhMDStRHbnSLh7Ypf36h/go-hamt-ipld"
-	"gx/ipfs/QmR8BauakNcBa3RbE4nbQu76PDiJgoQgz8AJdhJuiU4TAw/go-cid"
-	"gx/ipfs/QmRu7tiRnFk9mMPpVECQTBQJqXtmG132jJxA1w9A7TtpBz/go-ipfs-blockstore"
+	hamt "gx/ipfs/QmNf3wujpV2Y7Lnj2hy2UrmuX8bhMDStRHbnSLh7Ypf36h/go-hamt-ipld"
```
Comment (ZenGround0) [inline consensus/expected.go]: Blocking: these package names are redundant and should be changed back.

File: consensus/expected.go [hunk 2]
```diff
@@ -32,7 +35,7 @@ var (
 func init() {
 	ticketDomain = &big.Int{}
 	// The size of the ticket domain must equal the size of the Signature (ticket) generated.
-	// Currently this is a secp256.Sign signature, which is 65 bytes.
+	// Currently this is a secp256k1.Sign signature, which is 65 bytes.
```
Comment (ZenGround0) [inline consensus/expected.go]: 🙏

File: consensus/expected.go [hunk 3]
```diff
@@ -296,7 +299,8 @@ func (c *Expected) validateMining(ctx context.Context, st state.Tree, ts types.T
 
 // IsWinningTicket fetches miner power & total power, returns true if it's a winning ticket, false if not,
 //    errors out if minerPower or totalPower can't be found.
-//    See https://github.com/filecoin-project/aq/issues/70 for an explanation of the math here.
+//    See https://github.com/filecoin-project/specs/blob/master/expected-consensus.md
```
Comment (ZenGround0) [inline consensus/expected.go]: Thanks!

File: consensus/expected.go (hunk 1 above)
Comment (steven004) [reply consensus/expected.go]: Reversed back. Thanks for pointing out.

**Comment by steven004** [comment]
Changes made as per the request. Please review again.

**Pull request merged**